diff --git a/dist/_node-chunks/chunk-EGC27Z5J.js b/dist/_node-chunks/chunk-EGC27Z5J.js
index 9571ae8cde2f2f0d6322ea864d831cbba0d3a218..8750d86c754dcf5f0fcd066bdc7002ad4bbf42c7 100644
--- a/dist/_node-chunks/chunk-EGC27Z5J.js
+++ b/dist/_node-chunks/chunk-EGC27Z5J.js
@@ -733,7 +733,7 @@ async function importModule(path2, { skipCache = !1 } = {}) {
   }
   let mod;
   try {
-    let resolvedPath = win32.isAbsolute(path2) ? pathToFileURL2(path2).href : path2;
+    let resolvedPath = path2.startsWith("file:") ? new URL(path2).href : win32.isAbsolute(path2) ? pathToFileURL2(path2).href : path2;
     mod = await import(skipCache ? `${resolvedPath}?${Date.now()}` : resolvedPath);
   } catch (importError) {
     try {
